### PR TITLE
Cmtool usage updates

### PR DESCRIPTION
**Pull request by cmtool**

### CODE REVIEW

**Comment by cmtool** [comment]
This is the first time I've ever attempted to do a pull request on GitHub, so my apologies if I've messed this up. This is basically a bunch of spelling and grammar edits, along with changes to community.md that involve moving things that have been archived or depreciated to the right sections, fixing links.

The one area where feedback would be appreciated is the move the listservs to 'falling out of use'. I went through some of the archives, and it seems like discussion on the listservs is very slow, but I'd really welcome some feedback as to whether it's fine to label listservs as slowly being replaced by mattermost.

**Comment by cmtool** [comment]
Just saw some errors, so I closed this PR!

File: docs/api/usage.md [hunk 1]
```diff
@@ -51,8 +51,7 @@ array(
 )
 ```
 
-!!! note
-    A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
+*Note*: A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
```
Comment (mlutfy) [inline docs/api/usage.md]: I think the `!!! note` seems OK?

Capture d’écran de 2019-05-09 15-13-23

**Comment by mlutfy** [comment]
@cmtool It seemed OK to me? There was just the `!!! note` that I would personally have left as-is. I think you can continue to edit changes on a PR, instead of opening a new one?

Random example, if go to "Changes", then next to the file header:

Capture d’écran de 2019-05-09 15-17-08

**Comment by cmtool** [comment]
@mlutfy 

> @cmtool It seemed OK to me? There was just the `!!! note` that I would personally have left as-is. I think you can continue to edit changes on a PR, instead of opening a new one?

Haha, that's exactly the thing I was like 'oh no, it actually generates a thing that's really useful, oops'. XD

>Random example, if go to "Changes", then next to the file header:
Ah, okay, will do that! Thank you! :D

File: docs/api/usage.md (hunk 1 above)
Comment (cmtool) [inline docs/api/usage.md]:
```suggestion
!!! note
A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
```

File: docs/api/usage.md [hunk 2]
```diff
@@ -52,7 +52,7 @@ array(
 ```
 
 !!! note
-    A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
+A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
```
Comment (cmtool) [inline docs/api/usage.md]:
```suggestion
    A few specialized actions like `getsingle` or `getvalue` may return success in a different format.
```

**Comment by cmtool** [comment]
@mlutfy Okay, I went in and did two man changes. 

1. I added back in the !!! note
2. I moved IRC from ## Falling out of use to ## Out of use, based on the Mattermost documentation channel discussion where it was mentioned that CiviCRM no longer officially supports that channel.

Let me know if there are any further changes/edits you'd like me to make!

**Pull request merged**